src/components/GroupLinks.js

```diff
@@ -4,7 +4,11 @@ import PropTypes from "prop-types";
 const GroupLinks = ({ Website, Facebook, Twitter, Meetup }) => (
   <div className="links">
     {Website && (
-      <a href={Website} target="group" rel="nofollow" className="fas fa-link" />
+      <a href={Website}
```

Comment:
This is the code I referenced in https://github.com/LouisianaGroups/louisiana-groups/pull/11#issuecomment-377112555. We'll need to decide how to proceed, re: automatic code formatting.